vita3k/renderer/src/vulkan/pipeline_cache.cpp

```diff
@@ -172,7 +172,11 @@ void PipelineCache::read_pipeline_cache() {
 
 void PipelineCache::save_pipeline_cache() {
     size_t pipeline_size;
-    state.device.getPipelineCacheData(pipeline_cache, &pipeline_size, nullptr);
+    auto result = state.device.getPipelineCacheData(pipeline_cache, &pipeline_size, nullptr);
+    if (result != vk::Result::eSuccess) {
+        assert(false);
```

Comment:
Can you put LOG_ERRORs instead of assert_false ?

Reply:
i prefer to leave the assert false in case one of these goes unnoticed and its a lot easier to catch and debug

Reply:
Can you at least add a LOG_ERROR then? if these lines cause issues, it will most likely be driver specific and assert don't do anything in release builds.